python/GafferUI/Widget.py

```diff
@@ -604,6 +604,26 @@ def mousePosition( relativeTo=None ) :
 
 		return p
 
+	@staticmethod
+	def currentModifiers() :
+
+		return Widget._modifiers( QtWidgets.QApplication.queryKeyboardModifiers() )
+
+
+	@staticmethod
```

Comment:
Remove - this is just a clunky wrapper for `currentModifiers & Modifiers.Shift` etc.

Reply:
Ahh yeah, sorry hang over from when it was a `Set`.